picard/ui/metadatabox/__init__.py

```diff
@@ -282,19 +285,61 @@ def _copy_value(self):
                 except (TypeError, ValueError) as why:
                     log.warning(why)
                     value = ['']
-
-            if value is not None:
-                self.tagger.clipboard().setText(MULTI_VALUED_JOINER.join(value))
-                self.clipboard = value
+            return value
+
+        #TODO: Support  copying data from multiple files
+
+        items = self.selectedItems()
+        if len(items) > 1:
+            # We have multiple tags selected, so copy them as a JSON string
+            data = {}
+            for item in items:
+                key = self.tag_diff.tag_names[item.row()]
+                if key not in data:
+                    data[key] = {}
+
+                fieldnames = {self.COLUMN_TAG: 'tag_name', self.COLUMN_ORIG: 'original_value', self.COLUMN_NEW: 'new_value'}
+                fieldname = fieldnames[item.column()]
+
+                value = get_value_as_string(item)
+                if value and fieldname:
+                    data[key] = { fieldname: value } | data[key]
+            mimedata = QtCore.QMimeData()
+            mimedata.setData("application/json;contents=picard-tags", json.dumps(data).encode('utf-8'))
+            mimedata.setText(json.dumps(data))
```

Comment:
Maybe we should choose a different representation for plaintext. Something that is closer to just text. E.g.:

```
tagname1: value1
tagname2: value1; value2; value3
```

This at least can be easily reused by the user in a text editor. If another application would want to support this it would need to understand the structure anyway, at which point it could also use the `application/json;contents=picard-tags` data.

Reply:
Actually better suggesion: Use tab separated values. This would be compatible with spreadsheets and pate nicely there. So the above would be:

```
tagname1	value1
tagname2	value1	value2	value3
```

And when pasted to Google Sheets it is:

![grafik](https://github.com/user-attachments/assets/67cf4618-c788-45ec-97cd-8f0c9a813e6f)

This should work cross-platform for all common spreadsheet programs.

Reply:
Another thought: If we consider your original idea of having both original and new values and also look at the display in Picard, maybe having both original and new as separate column, and in each column the multiple values separated by `MULTI_VALUED_JOINER`, makes even more sense. Something like:

```
tagname1	old1; old2	new1; new2
tagname2	old1	new1; new2; new3
```

When copied over to a spreadsheet this would lead to a display very similar to what is inside Picard.

At this point I don't have a strong preference for this or my previous suggestion. @StevilKnevil @zas @Sophist-UK What are your thoughts?

Reply:
> As usual, we need to ensure strings don't contain separators or we need to escape them somehow.

That's a problem we already have at several places, including the existing copy and paste (pasting just splits at the joiner). It's probably outside the scope of this PR to fully address this.

Reply:
I've now implemented plain text as TSV. This works well in both Excel and Notepad++.

MOST importantly: I added some tests for this, so that I can iterate on it and keep my sanity :) Very much lowest bar for the testing at the moment, but something to build on. Would value any input on how "pythonic" my implementation is :)